bin/server.js

```diff
@@ -1,28 +1,105 @@
 #!/usr/bin/env node
-
 "use strict";
 
-var config, container, logger;
+/**
+ * Returns the usage information.
+ *
+ * @return {string}
+ */
+function usageMessage() {
+    return `Starts the OpenToken.io server.
+
+Accepts the following arguments:
+
+    --help -h -?               Shows this help message.
+    --override=FILE            Override settings in config.json with these.
+                               The properties are merged together.
 
-container = require("../lib/dependencies");
+Examples:
 
-// Initialize the server
-config = container.resolve("config");
-logger = container.resolve("logger");
+    # Starts the server using only config.json.
+    ./server.js
 
-// Allow overriding of the port
-if (process.env.PORT) {
-    logger.info(`Environment variable overriding port (was ${config.server.port}, is now ${process.env.PORT})`);
-    config.server.port = +process.env.PORT;
+    # Starts the server using config.json plus an override file.
+    ./server.js --override=sensitive-settings.json`;
 }
 
-if (process.env.DEBUG) {
-    logger.info("Environment variable DEBUG was set. DEBUG is on!");
-    config.debug = true;
+
+/**
+ * Loads the configuration
+ *
+ * @param {Dizzy} container
+ * @param {Object} args
+ * @return {Object} config
```

Comment:
Should this be a typedef? I'm good with object because it's pretty big, just thought I'd ask.